bindings/go/go.mod

```
@@ -2,3 +2,5 @@ module github.com/apple/foundationdb/bindings/go
 
 // The FoundationDB go bindings currently have no external golang dependencies outside of
 // the go standard library.
+
+require golang.org/x/xerrors v0.0.0-20190513163551-3ee3066db522 // indirect
```

Is this version still the one we want?

---

If I did this correctly, I think the newest one is:

```suggestion
require golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
```